src/hooks/useSidePanel.ts

```diff
@@ -14,13 +15,13 @@ import useResponsiveLayout from './useResponsiveLayout';
 import useWindowDimensions from './useWindowDimensions';
 
 /**
- * Hook to get the display status of the side pane
+ * Hook to get the display status of the Side Panel
  */
-function useSidePaneDisplayStatus() {
+function useSidePanelDisplayStatus() {
     const {isExtraLargeScreenWidth, shouldUseNarrowLayout} = useResponsiveLayout();
-    const [sidePaneNVP] = useOnyx(ONYXKEYS.NVP_SIDE_PANE);
+    const [sidePanelNVP] = useOnyx(ONYXKEYS.NVP_SIDE_PANEL);
     const [language] = useOnyx(ONYXKEYS.NVP_PREFERRED_LOCALE);
-    const [canUseHelpSidePanel = false] = useOnyx(ONYXKEYS.BETAS, {selector: (betas) => !!betas?.includes(CONST.BETAS.HELP_SIDE_PANEL)});
+    const [canUseHelpSidePanel = false] = useOnyx(ONYXKEYS.BETAS, {selector: Permissions.canUseHelpSidePanel});
```

Comment:
To confirm, we're doing this because using `usePermissions()` still made some tests fail?

Reply:
Yes, exactly. And I forgot to use `canUseHelpSidePanel` in the previous PR